synapse/http/server.py

```diff
@@ -490,11 +475,13 @@ class OptionsResource(resource.Resource):
     """Responds to OPTION requests for itself and all children."""
 
     def render_OPTIONS(self, request):
-        code, response_json_object = _options_handler(request)
+        request.setResponseCode(204)
+        request.setHeader(b"Content-Length", b"0")
+        request.setHeader(b"Cache-Control", b"no-cache, no-store, must-revalidate")
```

Comment:
yeah, I think we should remove this. If clients can cache OPTIONS responses, then they should.

Reply:
Less bytes then too! 🎉